src/main/java/codes/theotter/kixstar/command/KixstarWorldsCommand.java

```diff
@@ -29,28 +28,37 @@ public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command
             return false;
         }
 
-        Player player = Bukkit.getPlayer(args[1]);
+        UUID playerId = Bukkit.getPlayerUniqueId(args[1]);
+        if (playerId == null) {
+            this.sendMessage(sender, "&4Can't find " + args[1] + ". Do they exist?");
```

Comment:
You're being inconsistent here. At one place you're using String.format. But here you're not.